jni-glue/src/vm.rs

```diff
@@ -1,37 +1,43 @@
 use super::*;
 
 /// FFI: Use **&VM** instead of *const JavaVM.  This represents a global, process-wide Java exection environment.
-/// 
+///
```

Comment:
Aside, I think it would be nice if you RustFmt the code.

Reply:
I dislike what RustFmt does to my code.  Someday I'll be dragged into the modern age of automatic code formatters, but not today!